types/fundamental/real32quant/write.C

```diff
@@ -0,0 +1,96 @@
+#include <ROOT/RField.hxx>
+#include <ROOT/RNTupleModel.hxx>
+#include <ROOT/RNTupleUtil.hxx>
+#include <ROOT/RNTupleWriteOptions.hxx>
+#include <ROOT/RNTupleWriter.hxx>
+
+using ROOT::Experimental::EColumnType;
+using ROOT::Experimental::RField;
+using ROOT::Experimental::RNTupleModel;
+using ROOT::Experimental::RNTupleWriteOptions;
+using ROOT::Experimental::RNTupleWriter;
+
+#include <limits>
+#include <memory>
+#include <string_view>
+
+static constexpr double pi = 3.14159265358979323846;
+
+template <typename T>
+static std::shared_ptr<T> MakeFieldReal32Quant(RNTupleModel &model,
+                                               std::string_view name,
+                                               int nBits, double min, double max) {
+  assert(nBits >= 3 && nBits <= 32);
```

Comment:
I'm confused, should be
```suggestion
  assert(nBits >= 1 && nBits <= 32);
```
no? Is it possible to turn this into a construct that is actually checked?

Reply:
Sorry, I meant to change it to 1 but somehow I changed it to 3..